src/Nethermind/Nethermind.DataMarketplace.Consumers/Refunds/Services/RefundClaimant.cs

```diff
@@ -110,6 +102,19 @@ public async Task<RefundClaimStatus> TryClaimRefundAsync(DepositDetails deposit,
                 : RefundClaimStatus.Unconfirmed(transactionHash);
         }
 
+        private void DisplayRefundInfo(DepositDetails deposit, ulong now)
+        {
+            var timeLeftToClaimRefund = deposit.GetTimeLeftToClaimRefund(now);
+            if (timeLeftToClaimRefund > 0 && _logger.IsInfo)
```

Comment:
we don't mix logger logic with program logic -> make it into two ifs

Reply:
true, true